react_thervu_application/src/control/input/input.component.js

```diff
@@ -3,27 +3,27 @@ import PropTypes from 'prop-types';
 import classes from '../input/input.css';
 
 const Input = (props) => {
```

Comment:
don't put braces if there is only one parameter.
since it is a generic component, destructure the props in parameter section itself and use it..

Reply:
okay will destructure the props